src/client/api/init.c

```diff
@@ -50,8 +50,6 @@ static int		module_initialized;
 
 const struct daos_task_api dc_funcs[] = {
 	/** Management */
-	{dc_mgmt_svc_rip, sizeof(daos_svc_rip_t)},
-	{dc_pool_extend, sizeof(daos_pool_extend_t)},
```

Comment:
Are we also removing daos_svc_rip_t and daos_pool_extend_t? Or there are other users of them?